src/main/webapp/style.css

```diff
@@ -1,7 +1,59 @@
+body, html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.card {
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  transition: box-shadow .25s;
+  border-radius: 2px;
+  margin-top: 30px;
+  width: 500px;
+}
+
+.card-container {
```

Comment:
Can you put your CSS in a hierarchical order? I'm mixed up by things like this outer rule being in the middle of the card rules.

Reply:
(This also means I'd expect .srp and so forth to appear at the start.)